docs/user-guide/projects/project-lists/README.md

```diff
@@ -75,7 +85,7 @@ Projects can be filtered in OpenProject. The default view will list all currentl
 
 To adjust the project lists view use the **Filters** button, select the filtering criteria you require and click the blue **Apply** button.
 
-If you want to save this filtered list use the menu in the far right corner (three dots) and click **Save**.
+If you want to save this filtered list use the menu in the far right corner (three dots) and click **Save as**. Alternatively you can also use **Save as link** next to the warning message.
```

Comment:
Another thing to consider here is that this is not always **Save as**. There is two cases:

1. The modified a list that can be saved after the changes, then the link will be **Save** not **Save as**. And in the more icon button there will be both actions **Save** and **Save as**.
2. The user has modified a list that can't be saved with the changes (eg. a default project list like "Active projects"). Then there is only the option to **Save as**.

Reply:
@marcalcobe good point and important to document. However, changing persisted own lists was moved out of 13.4. (https://community.openproject.org/projects/openproject/work_packages/52144/)
I will add this once the feature is implemented